airflow/utils/db.py

```diff
@@ -1018,8 +1018,18 @@ def upgradedb(session: Session = NEW_SESSION):
         exit(1)
 
     with create_global_lock(session=session, lock=DBLocks.MIGRATIONS):
-        log.info("Creating tables")
-        command.upgrade(config, 'heads')
+        log.info("Running migrations")
+        rev_2_0_0_head = 'e959f08ac86c'
+        _lowerband = revision.split(':')[0] if ":" in revision else None
+        if _lowerband and _lowerband != rev_2_0_0_head:
+            sql = True  # Set to true if user passes a range of revisions
+            try:
+                command.history(config, rev_range=f"{rev_2_0_0_head}:{_lowerband}")
+            except Exception:
+                raise AirflowException(
+                    f"Supported revision is from {rev_2_0_0_head} which is airflow 2.0.0 head"
+                )
```

Comment:
This won't work if you try to have a revision range from (for example) 2.2.0->2.2.4 will it?

Reply:
It will work. It will first check if we can get history from 2.0.0 to 2.2.0 which will succeed, then it means the range is correct and upgrade command can run

Reply:
Need to fix a case of giving a single revision that's not a range...